src/components/OperationCard/OperationCard.tsx

```diff
@@ -27,7 +24,7 @@ interface OperactionCardProps {
 }
 
 const OperationCard: FC<OperactionCardProps> = ({ operation, title, className, icon, data, isLoading, isSuccess, isError, offPreloader = false }) => {
-
+    const currency = useAppSelector(state => state.persistedCurrencySlice.currency)
```

Comment:
useAppSelector<Here type of store>(state => state.persistedCurrencySlice.currency)